src/js/components/Footer.jsx

```diff
@@ -0,0 +1,58 @@
+import {grey600} from 'material-ui/styles/colors'
+import {footerLeftText, footerRightText} from '../../../config'
+import React, {Component} from 'react'
+
+const style = {
+  bottomLeft: {
+    display: 'inline-block'
+  },
+  bottomRight: {
+    float: 'right'
+  },
+  footer: {
+    alignItems: 'center',
+    background: grey600,
+    color: 'white',
+    display: 'flex',
+    fontSize: '0.8rem',
+    padding: '2px 5px'
+  },
+  spacer: {
+    flex: 1
+  }
+}
+
+const normalizeFooterText = (arg) => {
+  if (Array.isArray(arg)) {
+    return arg
+  }
+  if (typeof arg === 'string') {
+    return [arg]
+  }
+  
+  return ''
```

Comment:
Oh! Good catch. I see that now. I thought you were commenting on the order of the `if` statements and returns. Making the change.